Homework_6/Polindrome2.java

```diff
@@ -0,0 +1,35 @@
+package Homework_6;
+
+import java.util.Scanner;
+
+
+public class Polindrome2 {
+    public static boolean isNonNegative(int num) {
+
+        if (num >= 0)
+            return true;
+        else
+            return false;
+    }
+
+    public static void main(String[] args) {
+
+        String str, rev = "";
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter a string:");
+        str = sc.nextLine();
+        if (isNonNegative(Integer.parseInt(str)) == true) {
```

Comment:
You can simplify here the row to ** if (isNonNegative(Integer.parseInt(str))) {**